setup.py

```diff
@@ -8,9 +8,9 @@
 with open("README.md", "r") as fh:
     setup(
         name='j2v',
-        version='1.3.7',
-        author="Michal Zasadzinski",
-        author_email="michal.zasadzinski@gmail.com",
+        version='1.3.8',
+        author="Michal Zasadzinski, Ruthwik Ganesh",
```

Comment:
Should this be changed to `Cimpress Technology`? Over time, this will be a long list.